src/abstractdataframe/abstractdataframe.jl

```diff
@@ -614,7 +614,9 @@ unique!(df::AbstractDataFrame, cols)
 **Arguments**
 
 * `df` : the AbstractDataFrame
-* `cols` : the columns to compare – any valid DataFrames column specifier will work
+* `cols` :  column indicator (Symbol, Int, Vector{Symbol}, etc.)
+specifying the column(s) to compare. The first instance of each complete
```

Comment:
Good point. I have moved it down and tried to improve the wording, though it is still a bit wordy for my taste.